src/main/java/teamdraco/finsandstails/client/model/CrownedHorateeModel.java

```diff
@@ -0,0 +1,59 @@
+package teamdraco.finsandstails.client.model;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.resources.ResourceLocation;
+import software.bernie.geckolib3.core.event.predicate.AnimationEvent;
+import software.bernie.geckolib3.core.processor.IBone;
+import software.bernie.geckolib3.model.AnimatedGeoModel;
+import software.bernie.geckolib3.model.provider.data.EntityModelData;
+import teamdraco.finsandstails.FinsAndTails;
+import teamdraco.finsandstails.common.entities.CrownedHorateeEntity;
+
+public class CrownedHorateeModel extends AnimatedGeoModel<CrownedHorateeEntity> {
+	public static final ResourceLocation TEXTURE = new ResourceLocation(FinsAndTails.MOD_ID, "textures/entity/crowned_horatee.png");
+
+	@Override
+	public ResourceLocation getModelLocation(CrownedHorateeEntity entity) {
+		return new ResourceLocation(FinsAndTails.MOD_ID, "geo/entity/crowned_horatee.geo.json");
+	}
+
+	@Override
+	public ResourceLocation getTextureLocation(CrownedHorateeEntity entity) {
+		return TEXTURE;
+	}
+
+	@Override
+	public ResourceLocation getAnimationFileLocation(CrownedHorateeEntity entity) {
+		return new ResourceLocation(FinsAndTails.MOD_ID, "animations/entity/crowned_horatee.animations.json");
```

Comment:
It would be cleaner and slightly more efficient to make these static final resource locations, just like the texture.